concept/type/relationtype.feature

```diff
@@ -11,7 +11,12 @@ Feature: Concept Relation Type and Role Type
     Given connection has been opened
     Given connection does not have any database
     Given connection create database: typedb
-    Given connection opens schema transaction for database: typedb
+    Given connection open schema transaction for database: typedb
+
+    # TODO: `get label` and `set label` rename into `set name` and `get name` instead of label becasue it works ont with the whole label LA:BEL but name BEL in it! Do we want this reanme?
```

Comment:
Krishnan had a reasonable idea to refactor
`When relation(parentship) set label: marriage`
to
`When relation(parentship) set name: marriage`
Because `marriage` is a name, but its label is `parentship:marriage`. 
I've turned everything back to `set label` for simplicity, and I am ready to refactor everything globally.

If we want to get label, we ask for `label`, and its format is always `T:ROL`. To get `ROL` itself, it's name. Thoughts>

Reply:
Yes I think we can set `Name` for role we actually have this terminology in the 2.x codebase

Reply:
Cool! Done